### PR TITLE
Restructure Service folder system to allow for Unit testing

### PR DESCRIPTION
**Pull request by NanaKay007**
This PR introduces some important server-side changes. Kindly review extensively and ensure this branch runs fine on your local machine.

@Bendomey Need eagle eyes on this one

### CODE REVIEW

File: services/main/DTOs/CreateWaitlistInput.cs [hunk 1]
```diff
@@ -10,4 +10,22 @@ public class CreateWaitlistInput
 
     [RegularExpression("^(CLIENT|CREATOR)$")]
     public required string UserType { get; set; }
+
+    public override bool Equals(object? obj)
+    {
+        if (obj is not CreateWaitlistInput){
+            return false;
```
Comment (arkhurst) [inline services/main/DTOs/CreateWaitlistInput.cs:17]: Hey @NanaKay007 , not sure if you saw this but you have a couple of linting errors

Comment (NanaKay007) [reply services/main/DTOs/CreateWaitlistInput.cs:17]: ack'd. @Bendomey I could look into setting this up locally, but do you know how to do that? Kindly advise, thanks.

Comment (Bendomey) [reply services/main/DTOs/CreateWaitlistInput.cs:17]: I think linting was configured on a pre-commit hook. I'm guessing adding the unit test project might have moved somethings around, I'll pull branch locally and verify! =)

Comment (Bendomey) [reply services/main/DTOs/CreateWaitlistInput.cs:17]: Ah yeah, `main.sln` was moved, that's why. I can push a commit to change the path for accessing main.sln

Old: "services/main/main.sln"
New: "services/main.sln"

Lmk if I'm free to!

Comment (Bendomey) [reply services/main/DTOs/CreateWaitlistInput.cs:17]: So essentially, you dont have to run lint locally, once you commit, it automatically fixes all lint issues

**Comment by NanaKay007** [comment]
Yeah go for it bro!

Best,
Anikuabe Nana Kweku,
Swarthmore Class of 2022
LinkedIn <https://linkedin.com/in/anikuabenana> | Personal Website
<https://anikuabenana.com/>

**Review by Bendomey — APPROVED** [review]

**Review by Bendomey — COMMENTED** [review]
We probably have to add the test script to CI so that it passes before we merge? LMK what you think ;)

**Comment by Bendomey** [comment]
Will come back to this in the future after MVP is ready!

**Pull request closed without merging**